devel/spicy/autogen-spicy-docs

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
This file seems unused.

Reply:
This must be called manually as it needs access to files outside of the repository (similar to how it was inside the Spicy repo).

Reply:
Currently this scripts seems to be mentioned anywhere which is not great for making sure that the docs stay up to date. I am particular worried about changes landing in e.g., spicy-tftp and the Zeek docs becoming stale.

Over in zeek/spicy we [automated checking out the needed repos when building the docs](https://github.com/zeek/spicy/blob/30fb6309338b9bedd329c35c593fbad516748038/doc/Makefile#L64-L72), could we add something similar here? 

If that is not possible, the script should at least be called out prominently in `README`.

Reply:
Ok, I added that to the docs GH action. Not sure if that's the best place, but doing it Zeek-side isn't feeling great either.

Reply:
Well, never mind, that doesn't work because the script needs `spicy-doc` built. I'll leave the Makefile target in but remove from CI; and check if I can trigger it Zeek-side after all.